lib/tasks/assets.rake

```diff
@@ -84,8 +84,8 @@ namespace :assets do
     FileUtils.mv files_to_mv, target_path
   end
 
-  task :precompile_assets, [:external_asset_paths] => :environment do |_t, args|
-    cmd = "RAILS_ENV=#{Rails.env} EXTERNAL_ASSET_PATHS=#{args[:external_asset_paths]} rake assets:precompile"
+  task :precompile_assets, [:external_assets_path] => :environment do |_t, args|
+    cmd = "RAILS_ENV=#{Rails.env} EXTERNAL_ASSETS_PATH=#{args[:external_assets_path]} rake assets:precompile"
     puts "Running: #{cmd}"
```

Comment:
I'm leaving this here but as far as I can tell, we don't need this if we're invoking `rake assets:precompile` with both environment variables set (which is the case in CI at least).